### PR TITLE
Redux-in-components

### PR DESCRIPTION
**Pull request by Patfarmurs**

### CODE REVIEW

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @Patfarmurs , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Adding and removing books possible. Check status button works as expected
:heavy_check_mark: Linters all passing, Commit messages in imperative tense and first letters capitalized

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Great job working on this project. At the moment you have some broken links in your ReadMe file:
- The Live Demo Link. If you don't yet have a live link, you can use the text Coming Soon.
- The MIT license link gives error 404 when I click on it.
Kindly fix these two links.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -166,6 +166,6 @@ If you like this project...
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT] licensed.
```
Comment (levy002) [inline README.md:169]: - [ ] As requeted by the previous reviewer, Kindly make sure when we click on MIT in your Readme it directs us to the `MIT.md` [file](https://github.com/Patfarmurs/Book-store/blob/redux-components/MIT.md) which is the license of this project.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Patfarmurs   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Patfarmurs) [reply README.md:169]: Hi @levy002, the link is not working and I don't know why? which is why I added the MIT file Itself to solve the issue.

**Review by iambenkis — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @Patfarmurs 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: React set up
:heavy_check_mark: Redux
:heavy_check_mark: Linters passed 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me `@iambenkis` in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by Patfarmurs** [comment]
Thanks @iambenkis

**Pull request merged**